src/tests/utils/setup.ts

```diff
@@ -34,6 +37,15 @@ declare global {
    * @returns accessToken for authorization
    */
   function auth(userId: number) : string;
+
+
+  /**
+   * Update refresh token expiration date(refresh_token_expires_at) in the database with user sessions
```

Comment:
Wrong description for the function

Reply:
"execute request in database" is normal ?